bin/heudiconv

```diff
@@ -238,14 +231,25 @@ def embed_nifti(dcmfiles, niftifile, infofile, force=False):
     #orig_nii.update_header()
     #orig_nii.to_filename(niftifile)
     meta = ds.NiftiWrapper(new_nii).meta_ext.to_json()
-    with open(infofile, 'wt') as fp:
+    with open(infofile, 'at') as fp:
```

Comment:
that's not a valid json file i believe.

instead of appending text, read the json using `json.load` and then add the bids keys or read the bids and add the dcmmeta-keys. and then write it back out.